man/sysusers.d.xml

```diff
@@ -100,15 +103,16 @@
   <refsect1>
     <title>Configuration File Format</title>
 
-    <para>The file format is one line per user or group containing
-    name, ID, GECOS field description and home directory:</para>
+    <para>The file format is one line per user or group containing name, ID, GECOS
+    field description, home directory, and login shell:</para>
 
-    <programlisting>#Type  Name   ID              GECOS                 Home directory
-u      httpd  440             "HTTP User"
-u      authd  /usr/bin/authd  "Authorization user"
-g      input  -               -
-m      authd  input
-u      root   0               "Superuser"           /root</programlisting>
+    <programlisting>#Type Name     ID             GECOS                 Home directory Shell
+u     httpd    440            "HTTP User"
+u     authd    /usr/bin/authd "Authorization user"
+u     postgres -              "Postgresql Database" /var/lib/psql  /usr/libexec/postgresdb
+g     input    -              -
+m     authd    input
+u     root     0              "Superuser"           /root</programlisting>
```

Comment:
i figure the new column is particularly useful for the root user, too. maybe augment the example here and add /bin/zsh as shell for it? just to make a point?